index.js

```diff
@@ -21,24 +17,26 @@ function buildTargets(options) {
   return assign({}, defaultTargets, options.additionalTargets);
 }
 
-module.exports = function buildAirbnbPreset(context, options) {
+module.exports = declare(function buildAirbnbPreset(api, options) {
   var transpileTargets = (options && options.targets) || buildTargets(options || {});
 
   var debug = (options && typeof options.debug === 'boolean') ? !!options.debug : false;
 
+  api.assertVersion(7);
+
   return {
     presets: [
-      require('babel-preset-env').default(null, {
+      [require('@babel/preset-env'), {
         debug: debug,
         exclude: [
           'transform-async-to-generator',
-          'transform-es2015-template-literals',
+          'transform-template-literals',
           'transform-regenerator'
         ],
-        modules: false,
+        modules: options.modules,
```

Comment:
so presumably this defaults to "auto" or `true`, and someone passing `false` can override it - if this is `undefined`, what happens?

Separately, how does `auto` work? Can you link to docs?

Reply:
Looks like we didn't update the docs, sorry. Filed: https://github.com/babel/website/issues/1852

* `true`- error
* `false` - never compile ESM -> CJS
* `"commonjs"` - always compile ESM -> CJS
* `"auto"` - `false` if caller integration sets `supportsStaticESM: true` like Webpack, otherwise `"commonjs"`

Reply:
@ljharb Seems like if the key is set to `undefined`, it will still be treated like the key was not set. [It will just become `auto`](/babel/babel/blob/4249dbc8eac430b626a7d91362bfcc4be7e33957/packages/babel-preset-env/src/normalize-options.js#L142).